tests/core/test_state_sync.py

```diff
@@ -1426,16 +1426,16 @@ def test_rollback(state_sync: EngineAdapterStateSync, mocker: MockerFixture) ->
     state_sync.rollback()
     calls = {(a.sql(), b.sql()) for (a, b), _ in restore_table_spy.call_args_list}
     assert (
-        f'"{state_sync.schema}"."_snapshots"',
-        f'"{state_sync.schema}"._snapshots_backup',
+        f"{state_sync.schema}._snapshots",
```

Comment:
Is this right?

Reply:
Why would it be unquoted now?

Reply:
Because we used to mutate expressions in the engine adapter before. Please note that this is the `restore_table` call, not the final sql